conans/client/manager.py

```diff
@@ -282,24 +273,24 @@ def _get_graph_builder(self, loader, update, remote_proxy):
         graph_builder = DepsGraphBuilder(remote_proxy, self._user_io.out, loader, resolver)
         return graph_builder
 
-    def _get_deps_graph(self, reference, profile, filename, current_path, remote_proxy):
```

Comment:
All these methods do not require the current_path anymore, only used for the load_virtual